lib/web/apiserver.go

```diff
@@ -635,6 +635,7 @@ func (h *Handler) bindDefaultEndpoints() {
 	h.PUT("/webapi/sites/:site/databases/:database", h.WithClusterAuth(h.handleDatabaseUpdate))
 	h.GET("/webapi/sites/:site/databases/:database", h.WithClusterAuth(h.clusterDatabaseGet))
 	h.GET("/webapi/sites/:site/databases/:database/iam/policy", h.WithClusterAuth(h.handleDatabaseGetIAMPolicy))
+	h.GET("/webapi/scripts/database-access/configure/sqlserver/:token/configure-ad.ps1", httplib.MakeHandler(h.sqlServerConfigureADScriptHandle))
```

Comment:
FWIW, I wish we hadn't used `/desktop-access/` in the URL for the desktop scripts. We change product names/branding all the time, I would have preferred something more product-agnostic.

While it's too late for desktops, you can still avoid this and use something like `/webapi/scripts/databases/configure/...`